src/webportal/src/app/user/fabric/batchRegister/Table.jsx

```diff
@@ -92,6 +95,14 @@ export default function Table() {
     },
   };
 
+  const DropdownDisabledDiv = styled.div`
+    background-color: #ffffff;
+    border: 1px solid #a6a6a6;
+    margin-left: -12px;
```

Comment:
I think `styles={{title: {padding: 0}}}` could be used to clear dropdown's style.

Reply:
https://developer.microsoft.com/en-us/fabric#/controls/web/dropdown#IDropdownStyles

Reply:
Thanks for your suggestion. Fixed.